manager.go

```diff
@@ -695,7 +695,7 @@ func (m *jobManager) resolveImageOrVersion(imageOrVersion, defaultImageOrVersion
 		}
 	}
 
-	return "", "", fmt.Errorf("unable to find a release matching %q on https://openshift-release.svc.ci.openshift.org or https://origin-release.svc.ci.openshift.org", imageOrVersion)
+	return "", "", fmt.Errorf("unable to find a release matching %q on https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com or https://origin-release.apps.ci.l2s4.p1.openshiftapps.com", imageOrVersion)
```

Comment:
Why don't we have a cname redirect for this?

Reply:
It looks like the new cluster utilizes the cert-manager and has exposed these 2 ingresses...
Ocp:
https://amd64.ocp.releases.ci.openshift.org/
Origin:
https://amd64.origin.releases.ci.openshift.org/
I can update the URLs accordingly.

Reply:
Instead of making tihs change, let's get a DNS openshift-release.ci.openshift.org and origin-release.* created and pointed, then we can change these to that (in a separate PR).  The pull spec change is fine.

You need to change the chat bot to look up the release stream from app.ci, not api.ci, since api.ci is now out of date.

Reply:
https://github.com/openshift/release/pull/14635